sync_community_skills/skills.json

```diff
@@ -0,0 +1 @@
ativecommons-certificate", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "primary-site-includes", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "og-image-generator", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "wp-theme-openglam", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "cc-licenses-data", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "wp-plugin-vocabulary-blocks", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "wp-theme-cc-commoners", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "wp-plugin-cc-global-network", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}, {"name": "data-science-playground", "languages": [{"message": "API rate limit exceeded for 122.173.170.26. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)", "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}]}]
```

Comment:
@dhruvkb @zackkrida Here in languages of few repository an error message has been displayed since My Github api call limit has been reached but I can be fixed if we set `GITHUB_ADMIN_TOKEN` used in other packages of this repo.

Reply:
@TimidRobot Shall I redirect the token for `cc-creativecommons-github-io-bot` with the email specified in the other-packages.